lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -284,6 +301,44 @@ class LighthouseViewerReport {
   }
 
   /**
+   * Handles changes to the gist url input.
+   */
+  onInputChange(e) {
```

Comment:
I doubt anyone will hand-type the URL, but those that do will have to do as eric said.

for those on mobile devices that just paste the URL into the field, it'll trigger immediately after paste. (i just tested it out)